test/test_functional_tensor.py

```diff
@@ -56,6 +56,38 @@ def test_crop(self):
         cropped_img_script = script_crop(img_tensor, top, left, height, width)
         self.assertTrue(torch.equal(img_cropped, cropped_img_script))
 
+    def test_hue(self):
+        script_adjust_hue = torch.jit.script(F_t.adjust_hue)
+        fns = ((F.adjust_hue, F_t.adjust_hue, script_adjust_hue),)
+
+        for _ in range(20):
+            channels = 3
+            dims = torch.randint(1, 50, (2,))
+            shape = (channels, dims[0], dims[1])
+            shape = (3, 2, 2)
+            img = torch.randint(0, 256, shape, dtype=torch.uint8)
+
+            factor = torch.rand(1) - 0.5
+
+            img_clone = img.clone()
+            for f, ft, sft in fns:
+
+                ft_img = ft(img, factor)
+                sft_img = sft(img, factor)
+
+                img_pil = transforms.ToPILImage()(img)
+                f_img_pil = f(img_pil, factor)
+                f_img = np.array(f_img_pil, dtype=np.float)
+
+                diff = (ft_img.to(dtype=torch.float) - f_img.astype(float)).norm().numpy()
+                rel_err =  diff / np.linalg.norm(f_img.astype(float))
```

Comment:
Instead of returning the relative error by dividing the norm, which is a function of the number of elements, can you return as well the max absolute difference between the predicted value and the expected value?
```python
diff = (ft_img - f_img).abs().max()
```
?
In practice, I've found it to be useful to account of rounding errors, and it is informative of the max error that you are having. Ideally it should be 0, but due to numerical differences it could be slightly more as well.

Reply:
I tried this "diff". However i have seen that torch.fmod (used in rgb2hsv) and fmod have different precision. This results in ft_img and f_img differing by 1 (or 255). This creates issues with the testing.

Reply:
The main problem is that imgs are uint8 and a small difference (1 or 255) gets blown up. torch.fmod and math.fmod have different precision and when I was testing I saw rgb2hsv returning h values which differ from baseline by 1 or 2. This causes issues with the testing. Please let me know.

Reply:
The difference of 255 happens even if you cast to float before the subtraction in the test?
Also, is the difference due mainly to fmod or do you think it could come from somewhere else?

Reply:
No.  Before cast.  rgb2hsv fmod was the only place where i found this issue happens.

Reply:
So, `torch.fmod` should be similar to `fmod` in C. Would using `torch.remainder` fix the issue?

Reply:
I tried torch.remainder. I tried debugging  math.fmod vs torch.fmod there is a small difference which when multiplied by 255.0 and subsequently rounded is causing the discrepancy.